x/staking/client/cli/tx.go

```diff
@@ -257,12 +265,12 @@ func SignerUpdateTxCmd() *cobra.Command {
 	cmd.Flags().Uint64(FlagBlockNumber, 0, "--block-number=<block-number>")
 	cmd.Flags().Int(FlagNonce, 0, "--nonce=<nonce>")
 
-	cmd.MarkFlagRequired(FlagValidatorID)
-	cmd.MarkFlagRequired(FlagTxHash)
-	cmd.MarkFlagRequired(FlagNewSignerPubkey)
-	cmd.MarkFlagRequired(FlagLogIndex)
-	cmd.MarkFlagRequired(FlagBlockNumber)
-	cmd.MarkFlagRequired(FlagNonce)
+	cmd.MarkFlagRequired(FlagValidatorID)     //nolint
+	cmd.MarkFlagRequired(FlagTxHash)          //nolint
+	cmd.MarkFlagRequired(FlagNewSignerPubkey) //nolint
+	cmd.MarkFlagRequired(FlagLogIndex)        //nolint
+	cmd.MarkFlagRequired(FlagBlockNumber)     //nolint
+	cmd.MarkFlagRequired(FlagNonce)           //nolint
```

Comment:
We should also ignore err instead of `//nolint`, like -

```
_ = cmd.MarFlagXXX
```

Reply:
sure, I am relooking all the places I've made the changes again.